packages/lesswrong/lib/collections/users/schema.ts

```diff
@@ -941,6 +941,15 @@ const schema: SchemaType<"Users"> = {
     canCreate: 'guests',
     hidden: true,
   },
+  frontpageSelectedTab: {
+    type: String,
+    optional: true,
+    nullable: true,
```

Comment:
Hm... my guess is that this should not be nullable, and should have a default value of whatever the name of the latest posts tab is.

Reply:
(In fact I think we probably shouldn't include `lesswrong` in any of the tab names; we can just omit it, or if we want to specifically indicate that they aren't recombee tabs then we can say `forum-` instead.)